examples/platforms/nrf528xx/nrf52840/openthread-core-nrf52840-config.h

```diff
@@ -288,6 +288,16 @@
 #define OPENTHREAD_CONFIG_CSL_SAMPLE_WINDOW 30
 #endif
 
+/**
```

Comment:
I was thinking that, since this variable would effect the memory usage, we can configure different values on different platforms. For example, on some platforms we have very limited memory and we set this smaller. However, I think for `nrf52840` and `simulation` we can just use the default value in `mle.h`. I would remove the defines in `nrf52840` and `simulation`.